README.md

```diff
@@ -1,2 +1,67 @@
 # the-social-network-api
 An API / Backend for a social network that performs various functions on a NoSQL database (MongoDB).
```

Comment:
consider adding a little more detail to your description e.g. list the crud operations possible in your app